meson.build

```diff
@@ -75,6 +75,17 @@ install_data(
     install_dir : 'lib/systemd/system',
 )
 
+install_data(
+    'sources/openrc/__init__.py',
+    'sources/openrc/Openrc.py',
+    install_dir : get_option('libdir') / meson.project_name() / 'openrc',
+)
+
+install_data(
```

Comment:
Is it necessary to install this init script for everyone? 
Perhaps it would be cleaner to check if the system uses systemd or openrc.
But I don't know if this is a common practice.

Reply:
I think that's package manager should do. (think about gentoo -systemd use flag)

Reply:
And also, I do not know how to use and how to edit [meson.build](https://github.com/EmixamPP/linux-enable-ir-emitter/blob/master/meson.build), so, if you can, pls help me code  for that.

Reply:
No problem, I'll try to figure out how to handle that

In the next update, I plan to set all the variables present in `global.py` via meson, to facilitate the work of package manager who want to modify paths and things like that. 
I could also create two variables, SYSTEMD and OPENRC, which will be set to true if the system uses them. So all you'd have to do is check these variables to know which code to execute.

That's one possibility among many

Reply:
I had change directory structure, maybe you need to recheck the path in meson.build